hack/release.sh

```diff
@@ -41,12 +41,14 @@ Environments:
 
     REGISTRY                    container registry without repo name (default: quay.io/external_storage)
     VERSION                     if set, use given version as image tag
+    PULL_BASE_REF               if set, detect version from this git ref instead of "git describe"
     DOCKER_CONFIG               optional docker config location
     CONFIRM                     set this to skip confirmation
     ALLOW_UNSTABLE              by default, only master branch and tags that matches v<major>.<minor>.<patch> format are allowed, set this to skip this check (debug only)
     ALLOW_DIRTY                 by default, git repo must be clean, set this to skip this check (debug only)
     ALLOW_OVERRIDE              by default, stable image is not allowed to override, set this to skip (debug only)
     SKIP_BUILD                  set this to skip build phase (debug only)
+    SKIP_PUSH_LATEST            set this to skip pushing the latest stable image as the latest image
```

Comment:
[current prow job](https://github.com/kubernetes/test-infra/blob/2b47381ded875b75626ae9d5e3c3d6e5839a51e9/config/jobs/image-pushing/k8s-staging-sig-storage.sh) does not include `.git` directory, I add this to skip checking if the tag is the latest.

the `latest` tag is not recommended in production, I think it's fine not pushing it.